<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorDeckLinkInputWidget</class>
 <widget class="QWidget" name="InspectorDeckLinkInputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>196</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDevice">
     <item>
      <spacer name="horizontalSpacerDevice">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDevice">
       <property name="text">
        <string>&lt;a href=&quot;Device&quot; style=&quot;color: white;&quot;&gt;Device&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxDevice">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutFormat">
     <item>
      <spacer name="horizontalSpacerFormat">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelFormat">
       <property name="text">
        <string>&lt;a href=&quot;Format&quot; style=&quot;color: white;&quot;&gt;Format&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxFormat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTransition">
     <item>
      <spacer name="horizontalSpacerTransition">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>11</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTransition">
       <property name="text">
        <string>&lt;a href=&quot;Transition&quot; style=&quot;color: white;&quot;&gt;Transition&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTransition">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDuration">
     <item>
      <spacer name="horizontalSpacerDuration">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDuration">
       <property name="text">
        <string>&lt;a href=&quot;Duration&quot; style=&quot;color: white;&quot;&gt;Duration&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxDuration">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTween">
     <item>
      <spacer name="horizontalSpacerTween">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTween">
       <property name="text">
        <string>&lt;a href=&quot;Tween&quot; style=&quot;color: white;&quot;&gt;Tween&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTween">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDirection">
     <item>
      <spacer name="horizontalSpacerDirection">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDirection">
       <property name="text">
        <string>&lt;a href=&quot;Direction&quot; style=&quot;color: white;&quot;&gt;Direction&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxDirection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerMedia">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>labelDevice</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorDeckLinkInputWidget</receiver>
   <slot>resetDevice(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelFormat</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorDeckLinkInputWidget</receiver>
   <slot>resetFormat(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxDevice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorDeckLinkInputWidget</receiver>
   <slot>deviceChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxFormat</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorDeckLinkInputWidget</receiver>
   <slot>formatChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelTransition</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorDeckLinkInputWidget</receiver>
   <slot>resetTransition(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>-4</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTransition</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorDeckLinkInputWidget</receiver>
   <slot>transitionChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelDuration</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorDeckLinkInputWidget</receiver>
   <slot>resetDuration(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorDeckLinkInputWidget</receiver>
   <slot>durationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelTween</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorDeckLinkInputWidget</receiver>
   <slot>resetTween(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTween</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorDeckLinkInputWidget</receiver>
   <slot>tweenChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelDirection</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorDeckLinkInputWidget</receiver>
   <slot>resetDirection(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxDirection</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorDeckLinkInputWidget</receiver>
   <slot>directionChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>formatChanged(QString)</slot>
  <slot>deviceChanged(int)</slot>
  <slot>resetDevice(QString)</slot>
  <slot>resetFormat(QString)</slot>
  <slot>resetTransition(QString)</slot>
  <slot>transitionChanged(QString)</slot>
  <slot>resetDuration(QString)</slot>
  <slot>durationChanged(int)</slot>
  <slot>resetTween(QString)</slot>
  <slot>tweenChanged(QString)</slot>
  <slot>resetDirection(QString)</slot>
  <slot>directionChanged(QString)</slot>
 </slots>
</ui>
